src/main/resources/helptexts/doi.html

```diff
@@ -1,5 +1,17 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>Identifier</h1>
```

Comment:
This is the wrong helptext. Maybe there is no helptext for the DOI?

Reply:
Have not found a specific entree for DOI

Reply:
There isn't one

Reply:
for now, can you put in this?
```suggestion
<h1>Persistent Identifier</h1>
<p>
You can include this Digital Object Identifier (DOI) in your paper or electronic publication to refer to your dataset.
We also ask those who refer to your dataset in their publications to include this DOI. It contains a 'Persistent Identifier', which is coupled to your dataset and ensures that it will also remain discoverable via the Internet in the long term.

Note: This DOI will not yet work until the day after your dataset becomes available to the public.</p>
```

Reply:
done